definitions/ChrClasses.dbd

```diff
@@ -33,14 +33,14 @@ locstring Description_lang?
 locstring RoleInfoString_lang
 locstring DisabledString_lang
 int Field_9_0_1_34490_018?
-int<SpellVisualKit::ID> CharStartKit_SpellVisualKitID_Fallback?
-int<SpellVisualKit::ID> CharStartKit_Field_9_0_1_34615_004_Fallback?
-int<SpellVisualKit::ID> CharStartKit_SpellVisualKitID_Female_Fallback? // fall back to CharStartKit_SpellVisualKitID_Fallback if not set
-int<SpellVisualKit::ID> CharStartKit_Field_9_0_1_34615_004_Female_Fallback? // fall back to CharStartKit_Field_9_0_1_34615_004_Fallback if not set
-int Field_9_0_1_34821_023?
-int<SpellVisualKit::ID> CharStartKit_GroundSpellVisualKitID_Fallback?
-int<SpellVisualKit::ID> CharStartKit_Field_9_0_1_34821_009_Fallback?
-int CharStartKit_AnimLoopWaitTimeMs_Fallback?
+int<SpellVisualKit::ID> MaleCharacterCreationSpellVisualKitID?
```

Comment:
All these fields previously called `CharStartKit_xxx` (also in other tables) are fields that I manually verified to be fallbacks: It uses `CharStartKit_xxx` or if that's `0` the column here. Renaming just one is a bit weird: I would have expected these names to be the column names in `CharStartKit` and the ones in here still being named fallback or alike. Removing the connection altogether is weird.

Reply:
Yes, they are fallbacks for CharStartKit, just name gets kind of long at that point

Reply:
Also since I don't use/load CharStartKit, I didn't name them there

Reply:
Hm, that explains it. Seeing the client prefers `CSK` over `CC`/`CR` maybe you should load it at some point ;) 

I would really prefer if we kept the `CSK` names in sync with these, even if we drop the `CSK` prefix and/or `Fallback` suffix. My personal choice would be using the names currently used here (or drop the `SpellVisualKitID` as well as that's super redundant) in `CSK` and using `$nameFallback // fallback for CSK` here, i.e.

```
int<SpellVisualKit::ID> MaleCharacterCreationIdleVisual // Fall back to ChrClasses if not set.
```
in `CSK` and 
```
int<SpellVisualKit::ID> MaleCharacterCreationIdleVisualFallback
```
in `CC`/`CR`. That way one can still grep for the corresponding field, and the names are shorter.

Opinion(s)? // cc @Marlamin

Reply:
@bloerwald Your suggested names feel like the best option